slsim/Plots/lens_plots.py

```diff
@@ -36,31 +39,70 @@ def rgb_image(self, lens_class, rgb_band_list, add_noise=True):
         :param add_noise: boolean flag, set to True to add noise to the image, default
             is True
         """
-        image_r = simulate_image(
-            lens_class=lens_class,
-            band=rgb_band_list[0],
-            num_pix=self.num_pix,
-            add_noise=add_noise,
-            observatory=self._observatory,
-            **self._kwargs
-        )
-        image_g = simulate_image(
-            lens_class=lens_class,
-            band=rgb_band_list[1],
-            num_pix=self.num_pix,
-            add_noise=add_noise,
-            observatory=self._observatory,
-            **self._kwargs
-        )
-        image_b = simulate_image(
-            lens_class=lens_class,
-            band=rgb_band_list[2],
-            num_pix=self.num_pix,
-            add_noise=add_noise,
-            observatory=self._observatory,
-            **self._kwargs
-        )
-        image_rgb = make_lupton_rgb(image_r, image_g, image_b, stretch=0.5)
+        if self._observatory == "Roman":
+            # NOTE: Galsim is required which is not supported on Windows
+            image_r = simulate_roman_image(
+                lens_class,
+                band=rgb_band_list[0],
+                num_pix=self.num_pix,
+                oversample=5,
+                add_noise=add_noise,
+                **self._kwargs
+            )
+            image_g = simulate_roman_image(
+                lens_class,
+                band=rgb_band_list[1],
+                num_pix=self.num_pix,
+                oversample=5,
+                add_noise=add_noise,
+                **self._kwargs
+            )
+            image_b = simulate_roman_image(
+                lens_class,
+                band=rgb_band_list[2],
+                num_pix=self.num_pix,
+                oversample=5,
```

Comment:
here you use a oversample=5 argument. Is this differently treated for the other observatories? Maybe we can uniformly treat it such that it could also be used for others? Perhaps with the argument kwargs_numerics?

Reply:
Yes, in `simulate_roman_image`, the oversample is outside of `kwargs_numerics` because we want `image_model.image` to return the still oversampled image, and we do that by making an instance of the `SimAPI` class with `numpix = num_pix * oversample` and `pixel_scale = pixel_scale / oversample`. This is because we do the psf convolution using `galsim` instead of inside the `image_model` class.

It is still possible to put oversample in `kwargs_numerics` but then we have to do some gymnastics of extracting the oversample value and then changing it back to 1 after creating an instance of the SimAPI class so then it does not oversample again when we use `image_model.image`. I think it is better to just have the oversample parameter explicitly in the function arguments